### PR TITLE
[inductor] Move print_performance helper to utils

### PR DESCRIPTION
**Pull request by jansel**

### CODE REVIEW

File: torchinductor/utils.py [hunk 1]
```diff
@@ -52,3 +55,23 @@ def sympy_dot(seq1, seq2):
...
     return {id(x): x for x in it}.values()
+
+
+def timed(model, example_inputs, times=1):
+    synchronize()
+    torch.manual_seed(1337)
+    t0 = time.perf_counter()
+    for _ in range(times):
+        result = model(*example_inputs)
+        synchronize()
```
Comment (Chillee) [inline torchinductor/utils.py:66]: Should we synchronize after all runs are done?

Comment (Chillee) [reply torchinductor/utils.py:66]: Also, we could also perhaps just use triton's benchmarking code.

**Review by Chillee — APPROVED** [review]
lgtm

File: torchinductor/utils.py (hunk 1 above)
Comment (jansel) [reply torchinductor/utils.py:66]: It is useful to measure CPU overheads.  Plus this works for the C++ backend.  We use the gpu-only measurement in other places though.

**Pull request merged**